macros/sql_gen/build_metric_sql.sql

```diff
@@ -1,5 +1,6 @@
-{%- macro build_metric_sql(metric_dictionary, grain, dimensions, secondary_calculations, start_date, end_date, calendar_tbl, relevant_periods, calendar_dimensions, dimensions_provided) %}
+{%- macro build_metric_sql(metric_dictionary, metric_config, grain, dimensions, secondary_calculations, start_date, end_date, calendar_tbl, relevant_periods, calendar_dimensions, dimensions_provided) %}
```

Comment:
Oh now I'm doubly distressed about the inconsistent naming metric_config vs metric_config_dict